liberty-maven-webapp-parent/pom.xml

```diff
@@ -50,14 +50,12 @@
                             <goals>
                                 <goal>stop-server</goal>
                             </goals>
-                            <configuration>
-                                <failOnError>false</failOnError>
-                            </configuration>
                         </execution>
                         <execution>
                             <id>create-server</id>
                             <phase>prepare-package</phase>
                             <goals>
+                                <goal>install-server</goal>
```

Comment:
I will open an issue to remove the call to generate the xml file in create-server goal after confirming with Rajiv.